zeeguu/api/endpoints/search.py

```diff
@@ -212,3 +215,38 @@ def search_for_search_terms(search_terms, page: int = 0):
     article_infos = [UserArticle.user_article_info(user, a) for a in articles]
 
     return json_result(article_infos)
+
+@api.route(f"/{SUBSCRIBE_TO_EMAIL_SEARCH}/<search_terms>", methods=("GET",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@requires_session
+def subscribe_to_email_search(search_terms):
+    """
+    A user can subscribe to email updates about a search
+    """
+
+    search = Search.find(search_terms)
+    user = User.find_by_id(flask.g.user_id)
+    receive_email = True
+    SearchSubscription.update_receive_email(db_session, user, search, receive_email)
+    Search.update_receive_email(db_session, search_terms, receive_email)
```

Comment:
I do not understand this. Why does Search have a `receive_email` attribute? It should only be the Subscription that knows about this.

Reply:
I have added it to Search because you use search to show data in the UI. So if it is not added, I will not be able to see the changes made to the receive_email. Most endpoints use this, return json_result(search.as_dictionary()), they do not use search_subscription.

Reply:
Ah! In that case we should really refactor the UI to use Subscription instead of Search. I already observed that we should do that in the PR for the Web, but I thought we can delay it for after Wednesday. But in this case we should prioritize this straight away.

Reply:
Actually, we already have a meeting tomorrow @ 10am. We could use that too. But it might be a bit more than one hour.

Reply:
I would appreciate some help to fix this. Tomorrows meeting works fine for me, and I can do more than one hour.

Reply:
let's do that then!